include/radio-settings.h

```diff
@@ -136,6 +136,9 @@ void ofono_radio_settings_remove(struct ofono_radio_settings *rs);
 void ofono_radio_settings_set_data(struct ofono_radio_settings *rs, void *data);
 void *ofono_radio_settings_get_data(struct ofono_radio_settings *rs);
 
+void radio_set_rat_mode(struct ofono_radio_settings *rs,
```

Comment:
If you need to export this, please change the name to ofono_radio_set_rat_mode()